logdetective/server/models.py

```diff
@@ -254,6 +254,7 @@ class GeneralConfig(BaseModel):
     """General config options for Log Detective"""
 
     packages: List[str] = None
+    excluded_packages: List[str] = None
```

Comment:
This is sitting in my local checkout. I must have missed it on `git add` somehow. Oops.

The test I added didn't exercise the Pydantic paths, apparently.

Reply:
> The test I added didn't exercise the Pydantic paths, apparently.

true, my config mock was too much a mock 😔, I improved them this morning and verified that next time we will catch a similar error: https://github.com/fedora-copr/logdetective/pull/277